.github/actions/update-config/action.yaml

```diff
@@ -95,5 +95,6 @@ runs:
         gh pr merge "$PR_NUMBER" \
```

Comment:
revert the config change as well to trigger this action

Reply:
Done